### PR TITLE
[Spotless] Applying Google Code Format for several directories #15

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Applies spotless for several directories in project. Sets checkstyle to not fail on failure for several directories.
 
Note: adds directories that have no java in them. No negative impact, later will replace something that include entire project.

### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by pjfitzgibbons — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -89,6 +89,12 @@ spotless {
                     'prometheus/**/*.java',
                     'sql/**/*.java',
                     'common/**/*.java',
+                    'docs/**/*.java',
+                    'doctest/**/*.java',
+                    'relase-notes/**/*.java',
```
Comment (Yury-Fridlyand) [inline build.gradle:94]: There are no java files in there dirs
```suggestion
```

**Review by Yury-Fridlyand — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -89,6 +89,9 @@ spotless {
                     'prometheus/**/*.java',
                     'sql/**/*.java',
                     'common/**/*.java',
+                    'spark/**/*.java',
+                    'plugin/**/*.java',
+                    'sql/**/*.java',
```
Comment (MitchellGale) [inline build.gradle:94]:
```suggestion
```

**Review by Yury-Fridlyand — DISMISSED** [review]

File: build.gradle [hunk 3]
```diff
@@ -119,9 +121,8 @@ allprojects {
         sourceCompatibility = targetCompatibility = "11"
     }
     configurations.all {
-        resolutionStrategy.force "com.squareup.okio:okio:3.5.0"
-        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.9.0"
-        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.9.0"
+        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
+        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0"
```
Comment (Yury-Fridlyand) [inline build.gradle:125]: Please revert this

File: build.gradle [hunk 4]
```diff
@@ -119,9 +121,8 @@ allprojects {
...
     }
```
Comment (MitchellGale) [inline build.gradle:126]:
```suggestion
        resolutionStrategy.force "com.squareup.okio:okio:3.5.0"
        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.9.0"
        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.9.0"
    }
```

File: build.gradle (hunk 3 above)
Comment (MitchellGale) [reply build.gradle]: reverted

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by GumpacG — APPROVED** [review]

**Pull request merged**